### DISCUSSION TITLE
Is renovate support print external infomation from pakage?

### DISCUSSION THREAD

**Discussion by joebnb** [renovatebot/renovate#33199] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

gitlab

### Please tell us more about your question or problem

![image](https://github.com/user-attachments/assets/2ca62b3e-fe06-4439-a3f9-a0a07f0a8836)

in vscode there was a plugin called dependi it will shows alternative messages about upgrade package ,is renovate support this print on issue table or merge request

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by RahulGautamSingh** [comment]
I don't think we do

**Reply by rarkins** [reply]
If this comes from the npm `deprecationMessage` then we do

**Reply by joebnb** [reply]
> If this comes from the npm `deprecationMessage` then we do

https://registry.npmjs.com/@babel/plugin-proposal-class-properties 

yes, it's in meta data

**Reply by rarkins** [reply]
In that case yes, you should see information about such deprecated packages in the Dependency Dashboard issue

**Reply by joebnb** [reply]
i see but if a dependencyDashboardHeader template added, the deprecation warning will lost,is this warning move to  top of body template will better? and is the table shows up the content maybe more helpful or there have other by design cause

**Reply by joebnb** [reply]
> If this comes from the npm `deprecationMessage` then we do

i found that `deprecationMessage` is a variable that can be add in template, If this table could display the deprecation message of the metadata returned, it would be even better.
